### DISCUSSION TITLE
全局搜索能否让用户自己选择搜索结果顺序

### DISCUSSION THREAD

**Discussion by tcdy** [linuxdeepin/developer-center#3555] [Features Request & Ideas | 特性请求 & 头脑风暴]
默认是最佳匹配为第一位显示,但大多数情况下,最佳匹配并不是用户需要的,以我个人举例,最常用的情况是利用全局搜索打开应用,但是文件往往会排在前面,导致需要按方向键下键才能选中打开我要得应用,建议加上几个复选框让用户自主选择什么类型的文件排第一位,或者像utools那样,记住用户选择内容的频率,让用户经常选择的内容排在前面,类似于输入法的记忆功能,优先显示常用的内容

**Comment by BLumia** [comment]
> 但大多数情况下,最佳匹配并不是用户需要的

这个按我理解应该是与预期不符的吧。

@meiyixiang 看看？

**Comment by meiyixiang** [comment]
我认为这个是一个个性化的建议，用户自己可以设置搜索结果的排序类型，需求可以先记录下来。

**Comment by UTsweetyfish** [comment]
![截图_20230201140022](https://user-images.githubusercontent.com/124018391/215963995-cbb541e5-267a-4e56-b26d-d09c72497a00.png)
